src/SavedSearch.php

```diff
@@ -301,7 +301,7 @@ public function rawSearchOptions()
      */
     public function prepareSearchUrlForDB(array $input): array
     {
-        $taburl = parse_url(rawurldecode($input['url']));
+        $taburl = parse_url(html_entity_decode(rawurldecode($input['url'])));
```

Comment:
Are you using `html_entity_decode` to unsanitize the input ? In this case, you should use:
```suggestion
        $taburl = parse_url(rawurldecode(Sanitizer::unsanitize($input['url'])));
```

Reply:
ping @cconard96?